lib/ember_cli/path_set.rb

```diff
@@ -78,10 +78,18 @@ def bower
       end
     end
 
+    def bower_deps
```

Comment:
What do you think about changing the names of these methods to match the directories?

`bower_deps` => `bower_components`
`npm_deps` => `node_modules`?